numpy/polynomial/polynomial.py

```diff
@@ -325,7 +326,7 @@ def polymulx(c):
     return prd
 
 
-def polymul(c1, c2):
+def polymul(c1, c2, fourier=False):
```

Comment:
I don't have a strong opinion, but wonder whether we should inherit the terminology of `scipy.correlate` if we do this.  They use `method="direct"`, `"auto"`, and `"fft"`

The other question would be whether we should actually add it to convolve if we already use it in NumPy internally.  (I guess we used to say that scipy does it, so NumPy doesn't need to.)

Reply:
>The other question would be whether we should actually add it to convolve if we already use it in NumPy internally.

I'm not sure what you're asking. The function convolves by default. I just added the option to use Fourier transforms instead of convolution.

Reply:
I am saying that it may be plausible to add `concolve(..., method="fft")` if we need such an option here anyway.

Reply:
That could be an option, but it seems weird to have a convolve function with an option to not use convolution. A better idea would be to add an fftconvolve function like in scipy.

Reply:
`scipy.signal.correlate` has `method="..."` already. It would still return the convolution, just calculates it differently.
There is an ancient issue e.g. in gh-1858, there may be more.  I guess the main reason nobody we do not have it in NumPy is that we rather want to nudge users towards SciPy.  I am wondering if we should reconsider if we need it here.